api-lib/handleRefundGiveMsg.ts

```diff
@@ -0,0 +1,32 @@
+import assert from 'assert';
+
```

Comment:
We should probably figure out how to standardize filenames for this project, I really don't have a strong opinion but we have snake case for actions, then mixed lowerCamelCase and kebab-case in this package, and then CamelCase for types/classes (which I think is good). Anyway I'm open to ideas but a style guide might be nice. I've also struggled a bit with the db columns vs json vs var names. I have been using snake for db and lowerCamel for vars.

Reply:
I'll push a PR with the formatting we discussed today and see what @zashton thinks

Reply:
i changed to camel case for actions/input/response in my latest merged PR 😄